unskript-ctl/unskript_ctl_debug.py

```diff
@@ -26,50 +26,72 @@ def start_debug(args):
     """
     if not args:
         print("ERROR: Insufficient information provided")
-        return 
-    
+        return
+
     remote_config = args[0]
     remote_config = remote_config.replace('-','')
 
     if remote_config != "config":
         print(f"ERROR:The Allowed Parameter is --config, Given Flag is not recognized, --{remote_config}")
-        return 
+        return
     try:
         remote_config_file = args[1]
     except:
         print(f"ERROR: Not able to find the configuration to start debug session")
-        return 
-    
+        return
+
     if os.path.exists(remote_config_file) is False:
         print(f"ERROR: Required Remote Configuration not present. Ensure {remote_config_file} file is present.")
-        return 
-    
-    command = [f"openvpn --config {remote_config_file}"]
-    process = subprocess.Popen(command,
-                               stdout=subprocess.PIPE,
-                               stderr=subprocess.STDOUT,
-                               shell=True)
+        return
+
+    openvpn_log_file = "/tmp/openvpn_client.log"
+    command = [f"openvpn --config {remote_config_file} > {openvpn_log_file}"]
+    try:
+        process = subprocess.Popen(command,
+                                   stdout=subprocess.PIPE,
+                                   stderr=subprocess.PIPE,
+                                   shell=True)
+    except Exception as e:
+        print(f"ERROR: Unable to run the command {command}, error {e}")
+        return
 
     # Lets give few seconds for the subprocess to spawn
-    time.sleep(5)
+    try:
+        outs, errs = process.communicate(timeout=10)
+    except subprocess.TimeoutExpired:
+        # This is expected as the ovpn command needs to run indefinitely.
+        pass
```

Comment:
What happens when timeout happens, May be it is a good idea to print some error message?

Reply:
Timeout is expected as the process is running continuously